assets/sass/patterns/molecules/view-selector.scss

```diff
@@ -125,6 +125,7 @@
   }
 
   .view-selector__list-item {
+    display: none;
```

Comment:
what makes the new item visible then?

Reply:
https://github.com/elifesciences/pattern-library/blob/150c6f2af10fba2278e0addef7a21b8519f5bd3f/assets/sass/patterns/molecules/view-selector.scss#L138-L139 and https://github.com/elifesciences/pattern-library/blob/150c6f2af10fba2278e0addef7a21b8519f5bd3f/assets/sass/patterns/molecules/view-selector.scss#L144-L145

Reply:
Got it for the first 2 items, but I am missing how the `<li class="view-selector__list-item">` from the `otherLinks` part of the template get the same treatment, unless one of these classes is added through JS (but they seem too specific)?

Reply:
Pre-existing in the template: https://github.com/elifesciences/pattern-library/blob/150c6f2af10fba2278e0addef7a21b8519f5bd3f/source/_patterns/01-molecules/navigation/view-selector.mustache#L3-L10

(This is the part I've fought refactoring.)

Reply:
Figured it out: this is `none` because it's the selector for smaller screens and there the link disappears :man_facepalming: